### ISSUE TITLE
helm uninstall: erroneous exit code if no release found

### ISSUE THREAD

**Issue by DanielHabenicht** [helm/helm#9687]
If a release was not previously installed and `helm uninstall release-name` is run the following error occurs: 

```bash
Error: uninstall: Release not loaded: release-name: release: not found
```
That's fine and having read your design considerations I understand the reasoning behind the default behavior and it should not be altered. 

Looking at it from the automation perspective I would propose it as useful to add a flag similar to the `helm upgrade --install` flag. It would accept `release: not found` as successful uninstallation, as technically it was successfully uninstalled. 
Currently running the command in a CI pipeline errors if e.g. a previous installation wasn't successful and everything is teared down. 

Currently one has to use `|| true` but this has the drawback that it hides all errors, even those that might really happen during an uninstall. 

I have searched for similar issues but haven't found one, sorry if I might have overlooked it.

Output of `helm version`: `version.BuildInfo{Version:"v3.5.3", GitCommit:"041ce5a2c17a58be0fcd5f5e16fb3e7e95fea622", GitTreeState:"dirty", GoVersion:"go1.15.8"}`

**Comment by cndoit18** [comment]
It is useful, I have implemented it.

**Comment by davidema79** [comment]
Has this option been added? the PR (https://github.com/helm/helm/pull/9689) seemed to be reviewed, changed according to suggestions but never approved and merged. Currently with conflicts.

This feature has been implemented otherwise?

**Comment by DanielHabenicht** [comment]
no, not that I know

**Comment by remidebette** [comment]
Hi, this would be useful to have

**Comment by CTBT** [comment]
+ 1 waiting for this

**Comment by vakenbolt** [comment]
Agreed, this would be very useful!

**Comment by cndoit18** [comment]
same issue: https://github.com/helm/helm/issues/3352

**Comment by hamzdi** [comment]
same need please merge

**Comment by matthieu-dubin** [comment]
[Fix ](https://github.com/helm/helm/issues/3352) has be merged in V3.13.0.
